tests/e2e/chain.go

```diff
@@ -0,0 +1,83 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+
+	ojoapp "github.com/ojo-network/ojo/app"
+	"github.com/ojo-network/ojo/app/params"
+)
+
+const (
+	keyringPassphrase = "testpassphrase"
+	keyringAppName    = "testnet"
+)
```

Comment:
We have some duplicate consts here, could we pull from one source here & in client/tx/key.go?

Reply:
They actually are two separate set of constants. You could update one set and it would still work. One is for generating the keyring account for validators and one is initializing the keyring for the client (that might get abstracted out of Ojo at some point). I could make their names more specific?

Reply:
That works for me 👍